src/crypto/ossl/mod.rs

```diff
@@ -0,0 +1,45 @@
+//! OpenSSL implementation of cryptographic functions.
+
+use super::{CryptoError, SrtpProfile};
+
+mod cert;
+pub use cert::OsslDtlsCert;
+
+mod io_buf;
+mod stream;
+
+mod dtls;
+pub use dtls::OsslDtlsImpl;
+
+mod srtp;
+pub use srtp::OsslSrtpCryptoImpl;
+
+impl SrtpProfile {
+    // All the profiles we support, ordered from most preferred to least.
+    pub(crate) const ALL: &'static [SrtpProfile] =
```

Comment:
This and `keying_material_len` isn't OSSL specific and can be lifted to `super`

Reply:
Alright